nodejs-example-logs-api-extension/README.md

```diff
@@ -1,39 +1,46 @@
 # Example Logs API Extension in Node.js
+
 The provided code sample demonstrates how to get a basic Logs API extension written in Node.js 12 up and running.
 
 > Note: This extension requires the Node.js 12 runtime to be present in the Lambda execution environment of your function.
 
 There are two components to this sample:
+
 * `extensions/`: This sub-directory should be extracted to /opt/extensions where the Lambda platform will scan for executables to launch extensions
 * `nodejs-example-logs-api-extension/`: This sub-directory should be extracted to /opt/nodejs-example-logs-api-extension which is referenced by the `extensions/nodejs-example-logs-api-extension` executable and includes a nodejs executable along with all of its necessary dependencies.
 
 ## Prep Extension Dependencies
+
 Install the extension dependencies locally, which will be mounted along with the extension code.
 
 ```bash
-$ cd nodejs-example-logs-api-extension
-$ chmod +x index.js
-$ npm install
-$ cd ..
+cd nodejs-example-logs-api-extension
+chmod +x index.js
+npm install
+cd ..
 ```
 
 ## Layer Setup Process
+
 The extensions .zip file should contain a root directory called `extensions/`, where the extension executables are located and another root directory called `nodejs-example-logs-api-extension/`, where the core logic of the extension and its dependencies are located.
 
 Creating zip package for the extension:
+
 ```bash
-$ chmod +x extensions/nodejs-example-logs-api-extension
-$ zip -r extension.zip .
+chmod +x extensions/nodejs-example-logs-api-extension
+zip -r extension.zip .
```

Comment:
This line should be updated to only include the necessary files and directories, especially now that there are new files that are not needed for the extensions zip.